src/components/20-molecules/file-upload/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.9
+
+- Apply design-change suggestions from designers to allow coexistence of filename and per-file error message. (#1715)
+- Minor bugfixes.
+- Code is refactored and beautified.
```

Comment:
These 2 points are not necessary as raphi wrote. If you dont have a issue number for these bugfixes or nobody can see what you have fixed - remove that.

Reply:
Done (issue is actually for the bug fixes)